app/commands/forum/create_track_category.rb

```diff
@@ -0,0 +1,55 @@
+class Forum::CreateTrackCategory
```

Comment:
Let me know if you have a better suggestion for the name. 
I considered `SyncTrack`, `AddTrack` and `CreateTrack`.

Reply:
Instinctively I'd go `Track::CreateForumCategory` as Forum isn't really a primative in our world.